gengokit/handler/handler.go

```diff
@@ -241,11 +237,10 @@ func isValidFunc(f *ast.FuncDecl, m methodMap, pkgName string) bool {
 	return true
 }
 
-// mRecvTypeString returns accepts and *ast.FuncDecl.Recv recv, and returns the
+// mRecvTypeString accepts an *ast.FuncDecl.Recv recv, and returns the
```

Comment:
Hmmm... I did not even know why it was called that till I looked though the blames.

Seems it stood for `methodRecvTypeString`, I will change it to `recvTypeToString`.